### PR TITLE
feat: add "Apply for all" in component mapping alignment process

### PR DESCRIPTION
**Pull request by ma-zal**
## Change Log

- Added "Apply for all" in component mapping alignment process

### New dialog options:

image

### CODE REVIEW

File: src/local-development/align-components-mapping.ts [hunk 1]
```diff
@@ -209,17 +259,45 @@ export async function alignComponentsMapping(
...
+									break;
+								default:
+									throw new Error(`Unknown special answer "${userAnswer?.description}"`);
+							}
+						}
 						break;
+					}
 					case 'cloneAsNew':
-						selectedLocalComponentId = specialAnswers.CREATE_NEW_COMPONENT;
+						actionToProcess = { name: 'cloneAsNew' };
```
Comment (fuka-michal) [inline src/local-development/align-components-mapping.ts:300]: Should use some constant for option names 'cloneAsNew' is copy pasted too many times.

File: src/local-development/align-components-mapping.ts [hunk 2]
```diff
@@ -269,11 +351,10 @@ export async function alignComponentsMapping(
...
 								origin,
 							);
-							break;
 						}
-						default:
-							throw new Error(`Special answer "${selectedLocalComponentId?.description}" is unknown`);
-					}
+						break;
+					default:
+						throw new Error(`Unknown actionToProcess`);
```
Comment (fuka-michal) [inline src/local-development/align-components-mapping.ts:357]: But what action? Add ${actionToProcess.name}?

**Review by fuka-michal — APPROVED** [review]

File: src/local-development/align-components-mapping.ts (hunk 2 above)
Comment (ma-zal) [reply src/local-development/align-components-mapping.ts:357]: Done

File: src/local-development/align-components-mapping.ts (hunk 1 above)
Comment (ma-zal) [reply src/local-development/align-components-mapping.ts:300]: I am not a fan of `const CLONE_AS_NEW = 'cloneAsNew';` in this specific case. There is no special additional value.

**Pull request merged**